client/src/app/components/Header/Header.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import HamburgerMenu from "./MobileMenu/Hamburger";
+
+import "./header.css";
+
+class Input extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      inputBg: "grey"
+    };
+  }
+
+  render() {
+    return (
+      <div
+        className={`${
+          this.state.inputBg === "grey" ? "searchBarGrey" : "searchBarWhite"
+        } input-container`}
+      >
+        <input
+          onFocus={() => {
+            this.setState({ inputBg: "white" });
+          }}
+          onBlur={() => {
+            this.setState({ inputBg: "grey" });
+          }}
+          placeholder="Search for a project"
+          className={`${
```

Comment:
I see the same kind of logic on the parent className. That class toggling should be enough, and the child input will change it's style based on that (.input-container.open .search-bar {...})